addons/survey/models/survey_question.py

```diff
@@ -545,6 +545,7 @@ class SurveyQuestionAnswer(models.Model):
     _description = 'Survey Label'
 
     question_id = fields.Many2one('survey.question', string='Question', ondelete='cascade')
+    survey_id = fields.Many2one('survey.survey', related='question_id.survey_id')
```

Comment:
Not sure to find where it is used. I would avoid having it if possible to keep models as light as possible.